main/src/library/Order.flix

```diff
@@ -238,27 +238,22 @@ instance Order[Float64] {
 instance Order[BigDecimal] {
 
     pub override def less(x: BigDecimal, y: BigDecimal): Bool =
-        import java.math.BigDecimal.compareTo(BigDecimal): Int32 \ {};
-        (x `compareTo` y) < 0
+        (unsafe x.compareTo(y)) < 0
 
     pub override def lessEqual(x: BigDecimal, y: BigDecimal): Bool =
-        import java.math.BigDecimal.compareTo(BigDecimal): Int32 \ {};
-        (x `compareTo` y) <= 0
+        (unsafe x.compareTo(y)) <= 0
 
     pub override def greater(x: BigDecimal, y: BigDecimal): Bool =
-        import java.math.BigDecimal.compareTo(BigDecimal): Int32 \ {};
-        (x `compareTo` y) > 0
+        (unsafe x.compareTo(y)) > 0
 
     pub override def greaterEqual(x: BigDecimal, y: BigDecimal): Bool =
-        import java.math.BigDecimal.compareTo(BigDecimal): Int32 \ {};
-        (x `compareTo` y) >= 0
+        (unsafe x.compareTo(y)) >= 0
 
     pub def compare(x: BigDecimal, y: BigDecimal): Comparison =
-        import java.math.BigDecimal.compareTo(BigDecimal): Int32 \ {};
-        match x `compareTo` y {
+        match (unsafe x.compareTo(y)) {
```

Comment:
This could use the `Comparison.toComparison` (or whatever it is) helper function.